src/main/java/com/aerotrack/utils/clients/ryanair/RyanairClient.java

```diff
@@ -1,37 +1,53 @@
 package com.aerotrack.utils.clients.ryanair;
 
 import com.aerotrack.model.entities.Flight;
-import com.aerotrack.model.exceptions.ApiRequestException;
-import com.aerotrack.utils.clients.ApiClientWrapper;
 import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.json.JSONArray;
 import org.json.JSONObject;
+import retrofit2.Retrofit;
+import retrofit2.converter.scalars.ScalarsConverterFactory;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+import software.amazon.awssdk.services.s3.endpoints.internal.Value;
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Optional;
 
 @Slf4j
 @AllArgsConstructor
 public class RyanairClient {
-    private final ApiClientWrapper apiClientWrapper;
+
+    private final RyanairApiService ryanairApiService;
 
     public static RyanairClient create() {
-        return new RyanairClient(new ApiClientWrapper());
-    }
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("https://www.ryanair.com/api/booking/v4/it-it/")
```

Comment:
pls da ora in poi mettiamo en-EN ovunque
Poi credo che l'endpoint comune a tutte le richieste sia:
https://www.ryanair.com/api/

Reply:
era il tuo url